.github/workflows/release_staging.yml

```
@@ -0,0 +1,35 @@
+name: Deriv Api Docs Staging Workflow
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build_and_publish:
+    name: Builds and Publishes to Cloudflare Pages Staging
+    runs-on: Runner_16cores_Deriv-app # TODO: Replace this with the appropriate runner for Deriv-Api-Docs when provided
+    environment: Staging
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Node
+        uses: ./.github/actions/setup_node
+      - name: Install Dependencies
+        uses: ./.github/actions/npm_install_from_cache
+      - name: Build
+        uses: ./.github/actions/build
+        with:
+          CA: ${{ secrets.CA }}
+          CONTEXT_ARTIFACT_S3_BUCKET: ${{ secrets.CONTEXT_ARTIFACT_S3_BUCKET }}
```

```suggestion
          CONTEXT_ARTIFACT_S3_BUCKET: ${{ secrets.CONTEXT_ARTIFACT_S3_BUCKET }}
```

Not sure yet where this is used, so I'll leave it here for now